### DISCUSSION TITLE
9.2 HA Os

### DISCUSSION THREAD

**Discussion by black5cat** [home-assistant/operating-system#2197] [Show and tell]
Is it possible to turn off the system from the button in os home assistant ? I used to use a mini personal computer Msi cubi as a server, and it was convenient to turn off with a button and transfer, or temporarily turn off the system, now this is not and turning on is extremely inconvenient …

**Comment by agners** [comment]
There have been specifically the opposite requests in the past: Avoid shutting down when pressing the power button :sweat_smile: 

Currently systemd is configured to shutdown the system on a long press (5 seconds):
https://github.com/home-assistant/operating-system/blob/rel-9/buildroot-external/rootfs-overlay/etc/systemd/logind.conf#L22

**Comment by black5cat** [comment]
previously, it was possible to turn off from the system. it was bad, but it was. and now there is no way to stop the server without problems. to turn off the power. working with him .... I understand that earlier with other options, maybe it was safer.... now I do not know how to turn off the server at all without problems with an ssd disk, for example. or the loss of files. 😒

And thanks, I'll read and try the options. one question- can the file be customized to suit your interests? and is there a description for this setting. or is it better not to experiment🤔

**Reply by agners** [reply]
You can do the following in the CLI:

```
ha host shutdown
```

**Comment by black5cat** [comment]
long press on the power - just turns it off... or I didn't understand something. but this does not make the PC stop correctly?

**Reply by agners** [reply]
Hm, yeah I see on PC that is probably hardware/BIOS controlled, so the systemd configuration does nothing. It does work on Yellow, long-pressing the blue button.

So the above mentioned CLI command is the safe option.

**Comment by black5cat** [comment]
an early PC short press turned off the system guaranteed correctly.. is it possible to return the function only for pc? not for alternative installations of pi raspbbery . or other options . pc only.....
